pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1440,6 +1440,16 @@ type ObjectStoreSpec struct {
 	// +nullable
 	Gateway GatewaySpec `json:"gateway"`
 
+	// The protocol specification
+	// +optional
+	// +nullable
```

Comment:
Not nullable because not a pointer to a struct.

Reply:
Addressed in 556765ceea